### DISCUSSION TITLE
resume cursor is not visible on skins with reaklly dark cursor

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22209] [Q&A]
### tldr: I am aware this is an edgecase, but it should still be adressed, since cursors like this are possible, and there is no reason to scale the "orangeness" of resume cursor to the skins brightness
---

#### When I was still using stable, I made a skin to be used with gamma boosting for high AR reading, and made the cursor really dark for it to not half blind you. When I used this Skin in lazer (without the gamma boost just for fun and because my reading has improved), and paused and resumed my game I ran into this funny issue, where you can't see the resume cursor. In stable with the same skin the resume cursor is the default orange thing and always visble.

My suggested fix would be to use a static texture for the resume cursor thats always orange, and always visble no matter what.

Here are two videos demonstrating the issue as I encountered it, ill add the cursor files so you may easily reproduce it. 

Note that the cursor is not actually black rgb 0,0,0 but 8,8,8 , so you can see something in the gamma boosted video. I also uploaded the 8,8,8 version of the cursor.

https://user-images.githubusercontent.com/87947227/212520237-c871094d-8f9f-4ea6-93d9-cc84f3f1f9bd.mp4

https://user-images.githubusercontent.com/87947227/212520240-10915fe2-fb4e-4284-904d-e701fe1607f9.mp4

---
skin elements
---

![cursor](https://user-images.githubusercontent.com/87947227/212520255-8ff8a529-4f53-4f81-b9c7-2ddb11a3701a.png)
![cursor@2x](https://user-images.githubusercontent.com/87947227/212520256-fcd9425d-3234-4ff3-a085-da4a9bb1f87b.png)
![cursortrail](https://user-images.githubusercontent.com/87947227/212520258-483ccacc-2ead-46e8-aa75-cee62a7f87dd.png)

Btw I also think that the resume cursor should be a minimum large size so this stuff doesnt happen.

![image](https://user-images.githubusercontent.com/87947227/212520712-78e93ded-1096-452f-99e3-d27c32170769.png)

(resume cursor is within the "u" in the word cursor and you can go search it for a while if you dont remember where you paused)

**Comment by mk56-spn** [comment]
the reason this is happening is likely the same as https://github.com/ppy/osu/discussions/19562#discussioncomment-3314262

**Reply by Inconsist** [reply]
It seems similar, but differs in the case that a users only expectation here is to see a resume cursor. In that case the fix for this could still be to just make the resume cursor its own element. seems to be the easiest fix.
